edgedb/protocol/codecs/codecs.pyx

```diff
@@ -84,7 +84,7 @@ cdef class CodecsRegistry:
                 els = <uint16_t>hton.unpack_int16(frb_read(spec, 2))
                 for i in range(els):
                     frb_read(spec, 1)
-                    str_len = <uint16_t>hton.unpack_int16(frb_read(spec, 2))
+                    str_len = <uint32_t>hton.unpack_uint32(frb_read(spec, 4))
```

Comment:
You don't need to cast anymore.  That was the whole reason to add `uint` unpack variants, wasn't it?

Reply:
Yes. You're right. There are so many casts in the code that they look like a convention here :)